modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/00-list-type-definition.batch-engine-data.json

```diff
@@ -14,6 +14,104 @@
 		"version": "v1.0"
 	},
 	"items": [
+		{
+			"externalReferenceCode": "L_COMMERCE_RETURN_ITEM_STATUSES",
```

Comment:
Are all these status managed in the typescript component that print the "label"?